packages/experimental/Expander/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@fluentui-react-native/experimental-expander",
+  "version": "0.1.0",
+  "description": "A cross-platform Native Expander component using the Fluent Design System. Currently only implemented on windows",
+  "license": "MIT",
+  "author": "Microsoft <fluentuinativeowners@microsoft.com>",
+  "homepage": "https://github.com/microsoft/fluentui-react-native",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "lint": "fluentui-scripts eslint",
+    "depcheck": "fluentui-scripts depcheck",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor",
+    "windows": "react-native run-windows"
+  },
+  "dependencies": {
+    "@fluentui-react-native/component-cache": "^1.2.0",
+    "@fluentui-react-native/framework": "0.5.15"
+  },
+  "devDependencies": {
+    "@types/react-native": "^0.63.0",
+    "@uifabricshared/build-native": "^0.1.1",
+    "@uifabricshared/eslint-config-rules": "^0.1.1",
+    "react": "16.13.1",
+    "react-native": "^0.63.4",
+    "react-native-windows": "^0.63.0-0"
```

Comment:
We typically don't want to specify pre-release (`-x`) versions as dependencies

Reply:
I think I/we may have told her to use this syntax so that she can get both 0.63 and 0.64, since `^0.63.0` resolves to `>=0.63.0 <0.64.0` (https://devhints.io/semver). This syntax should resolve to `>=0.63.0 < 1`

That being said, having `react-native-windows` as a peer dependency should solve the issue and we can maybe remove the "-0", since the app consuming this component will bring it's own version of `react-native/react-native-windows`, regardless of what the package.json specifies.